docs/UI_CUSTOMIZATION.md

```diff
@@ -0,0 +1,133 @@
+    <item name="android:background">#FF00FF</item>
+    <item name="android:textSize">16sp</item>
+</style>
+```
+
+## Adding dark mode support
+Out of the box the SDK doesn’t support dark mode, but you can easily add this with some additional setup. Copy the base `AdyenCheckout` style in your `styles.xml` and change the parent to a Material DayNight theme. Now you can use the `values-night` resource folder to define colors and styles to be used in dark mode.
+
+```XML
+<style name="AdyenCheckout" parent="Theme.MaterialComponents.DayNight">
+    <!-- These color values are both defined in values/colors.xml and values-night/colors.xml to change the color based on selected mode -->
+    <item name="colorPrimary">@color/color_primary</item>
+    <item name="colorOnPrimary">@color/color_on_primary</item>
+    <item name="colorAccent">@color/color_accent</item>
+    <item name="android:colorBackground">@color/color_background</item>
+    <item name="colorBackgroundFloating">@color/color_background</item>
+    <item name="colorOnBackground">@color/color_on_background</item>
+    <item name="colorSurface">@color/color_surface</item>
+    <item name="colorOnSurface">@color/color_on_surface</item>
+
+    <item name="android:textColor">@color/text_color_primary</item>
+    <item name="android:textColorPrimary">@color/text_color_primary</item>
+    <item name="android:textColorSecondary">@color/text_color_primary</item>
+    <item name="android:textColorTertiary">@color/text_color_primary</item>
+    <item name="android:textColorLink">@color/text_color_link</item>
+    <item name="bottomSheetDialogTheme">@style/AdyenCheckout.BottomSheetDialogTheme</item>
+</style>
+```
+
+To learn more about Material dark theming you can read [this guide](https://m2.material.io/develop/android/theming/dark) or take a look at [our example app](https://github.com/Adyen/adyen-android/blob/main/example-app/src/main/res/values/styles.xml).
+
+## Overriding string resources
+
+It is possible to change text in the SDK by overriding string resources. To override a string resource you have to copy the string resource in your own strings.xml. The easiest way to find the string resource is to search for the string on Github. In case your app supports multiple languages you can do the exact same thing, but make sure to use the right `values` directory (for example `values-nl-rNL`).
+
+```XML
+<resources>
+    <!-- Original text is: "Change Payment Method" -->
+    <string name="change_payment_method">More Payment Methods</string>
+</resources>
+```
+
+Payment method names in the payment method list can be overridden with a configuration object:
+
+```kotlin
+DropInConfiguration.Builder(shopperLocale, environment, clientKey)
+    .overridePaymentMethodName(PaymentMethodTypes.SCHEME, "Credit cards")
+    .overridePaymentMethodName(PaymentMethodTypes.GIFTCARD, "Specific gift card")
+    .build()
+
+CheckoutConfiguration(shopperLocale, environment, clientKey) {
```

Comment:
Let's move the 2nd code sample to the configuration PR?
And also we can make it clear that you need one of these code samples not both, maybe put an `OR` between them or split them in 2 code blocks?